MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -25,13 +25,21 @@
 		16E3625C201265D600DF0592 /* ImageDownloadOperationSpy.swift in Sources */ = {isa = PBXBuildFile; fileRef = 16E3625B201265D600DF0592 /* ImageDownloadOperationSpy.swift */; };
 		16EF6C1E21193A9600AA580B /* CarPlayManagerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 16EF6C1D21193A9600AA580B /* CarPlayManagerTests.swift */; };
 		16EF6C22211BA4B300AA580B /* CarPlayMapViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 16EF6C21211BA4B300AA580B /* CarPlayMapViewController.swift */; };
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
These merge conflict markers are still in the codebase and are preventing Xcode from being able to open or compile the project:

```
xcodebuild: error: Unable to read project 'MapboxNavigation.xcodeproj'.
	Reason: The project ‘MapboxNavigation’ is damaged and cannot be opened due to a parse error. Examine the project file for invalid edits or unresolved source control conflicts.
```

The file must’ve gotten `git add`ed without manually resolving the merge conflicts.

Reply:
Corrected in my last commit